### PR TITLE
Topic tuneup - design folder -  GH 768

### PR DESCRIPTION
**Pull request by leslietilling**
## Purpose of this pull request

This pull request (PR) is the clean up effort for #768  and addresses the topics in the `design` folder. Upon walkthrough of the procedures, some screens were also added or updated as needed (also getting rid of the "cartoon balloons". Also the tuneup basics:

- Captions using sentence case
- Headings using sentence case and not infinitives
- Consistency in numbered step structure and style
- Edits for clarity

Will be forward ported to the 2.4 guide.

Internal staging build at /14/user-guide/v2.3/

## Magento release version

- [ ] 2.4.x

   Specify a patch release number, if applicable:

- [x] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [ ] Magento Open Source only
- [ ] Magento Commerce only

Is this update specific to an installed feature extension

- [ ] B2B extension
- [ ] Other feature set, please specify:

## Affected documentation pages

https://docs.magento.com/user-guide/v2.3/design/configuration.html
https://docs.magento.com/user-guide/v2.3/design/copyright-notice.html
https://docs.magento.com/user-guide/v2.3/design/demo-notice.html
https://docs.magento.com/user-guide/v2.3/design/design-menu.html
https://docs.magento.com/user-guide/v2.3/design/design-theme.html
https://docs.magento.com/user-guide/v2.3/design/favicon.html
https://docs.magento.com/user-guide/v2.3/design/footer.html
https://docs.magento.com/user-guide/v2.3/design/header.html
https://docs.magento.com/user-guide/v2.3/design/html-head.html
https://docs.magento.com/user-guide/v2.3/design/layout-update-place-block.html
https://docs.magento.com/user-guide/v2.3/design/layout-updates.html
https://docs.magento.com/user-guide/v2.3/design/logo-upload.html
https://docs.magento.com/user-guide/v2.3/design/merge-css.html
https://docs.magento.com/user-guide/v2.3/design/merge-javascript.html
https://docs.magento.com/user-guide/v2.3/design/page-layout-examples.html
https://docs.magento.com/user-guide/v2.3/design/page-layout-standard.html
https://docs.magento.com/user-guide/v2.3/design/page-layout.html
https://docs.magento.com/user-guide/v2.3/design/page-setup.html
https://docs.magento.com/user-guide/v2.3/design/schedule.html
https://docs.magento.com/user-guide/v2.3/design/storefront-branding.html
https://docs.magento.com/user-guide/v2.3/design/theme-assets.html
https://docs.magento.com/user-guide/v2.3/design/theme-default.html
https://docs.magento.com/user-guide/v2.3/design/theme-install.html
https://docs.magento.com/user-guide/v2.3/design/themes.html
https://docs.magento.com/user-guide/v2.3/design/welcome-message.html

### CODE REVIEW

File: src/design/merge-css.md [hunk 1]
```diff
@@ -2,21 +2,21 @@
...
 {:.bs-callout-info}
 CSS files can be merged only when working in [Developer Mode]({% link magento/installation-modes.md %}).
 
-## To merge CSS files:
+## Merge your CSS files
 
 1. On the _Admin_ sidebar, go to **Stores** > _Settings_ > **Configuration**.
 
-1. In the panel on the left under **Advanced**, choose **Developer**.
+1. In the left panel, expand **Advanced** and choose **Developer**.
```
Comment (hickskenh) [inline src/design/merge-css.md:14]:
```suggestion
1. In the left panel, expand ![]({% link images/images/btn-expand.png %}) **Advanced** and choose **Developer**.
```

File: src/design/merge-javascript.md [hunk 2]
```diff
@@ -2,21 +2,21 @@
...
 {:.bs-callout-info}
 JavaScript files can be merged only when working in [Developer Mode]({% link magento/installation-modes.md %}).
 
-## To merge JavaScript files:
+## Merge your JavaScript files
 
 1. On the _Admin_ sidebar, go to **Stores** > _Settings_ > **Configuration**.
 
-1. In the panel on the left under **Advanced**, choose **Developer**.
+1. In the left panel, expand **Advanced** and choose **Developer**.
```
Comment (hickskenh) [inline src/design/merge-javascript.md:14]:
```suggestion
1. In the left panel, expand ![]({% link images/images/btn-expand.png %}) **Advanced** and choose **Developer**.
```

File: src/design/page-layout.md [hunk 3]
```diff
@@ -2,26 +2,73 @@
...
+   ![]({% link images/images-ee/config-general-web-default-layouts.png %}){: .zoom}
+   [*Default Layouts*]({% link configuration/general/web.md %}#default-layouts)
 
-    ![]({% link images/images-ee/config-general-web-default-layouts.png %}){: .zoom}
-    [*Default Layouts*]({% link configuration/general/web.md %})
+1. Choose the **Default Product Layout** that you want to use for product pages.
+
+   This determines the layout that is used by default for product pages.
+
+   |No layout updates |By default, layout updates are not available for product pages.|
```
Comment (hickskenh) [inline src/design/page-layout.md:27]: is "By default," needed for each of theses.   the next table doesn't.

File: src/design/theme-assets.md [hunk 4]
```diff
@@ -2,24 +2,22 @@
...
-_Theme Assets_
+_Theme assets_
 
-## To sign static files:
+## Add a digital signature to static file URLs
 
 1. On the _Admin_ sidebar, go to **Stores** > _Settings_ > **Configuration**.
 
-1. In the panel on the left under **Advanced**, choose **Developer**.
+1. In the left panel, expand **Advanced** and choose **Developer**.
```
Comment (hickskenh) [inline src/design/theme-assets.md:18]:
```suggestion
1. In the left panel, expand ![]({% link images/images/btn-expand.png %}) **Advanced** and choose **Developer**.
```

File: src/design/theme-install.md [hunk 5]
```diff
@@ -2,25 +2,23 @@
...
+1. Download the package
+1. [Install the theme][3]
 
-To apply the theme to your store, see: [Using the Default Theme]({% link design/theme-default.md %}).
+To apply the theme to your store, see [Using the Default Theme]({% link design/theme-default.md %}).
 
 [1]: http://devdocs.magento.com/guides/v2.3/frontend-dev-guide/bk-frontend-dev-guide.html
 [2]: https://devdocs.magento.com/guides/v2.3/install-gde/prereq/connect-auth.html
-[3]: http://docs.magento.com/marketplace/user_guide/quick-tour/welcome.html
+[3]: https://devdocs.magento.com/guides/v2.3/frontend-dev-guide/themes/theme-install.html
```
Comment (hickskenh) [inline src/design/theme-install.md:24]: the text refers to create or modify, but the link goes to "install third party theme".   recommend using the parent topic.   https://devdocs.magento.com/guides/v2.3/frontend-dev-guide/themes/theme-overview.html

File: src/design/themes.md [hunk 6]
```diff
@@ -4,13 +4,13 @@ redirect_from:
...
+A responsive theme adjusts the page layout to fit the view port of the device. The sample _Luma_ theme has a flexible, responsive layout that can be viewed from the desktop, tablet, or mobile device.
 
-Magento themes include layout files, template files, translation files, and skins. A skin is a collection of supporting CSS, images, and JavaScript files that together, create the visual presentation and interactions that your customers experience when they visit your store. Themes and skins can be modified and customized by a developer or designer who has knowledge of Magento theme design and access to your server. To learn more, see the [Frontend Developer Guide][1].
+Magento themes include layout files, template files, translation files, and skins. A skin is a collection of supporting CSS, images, and JavaScript files that together, create the visual presentation and interactions that your customers experience when they visit your store. Themes and skins can be modified and customized by a developer or designer who has knowledge of Magento theme design and access to your server. To learn more, see our [Frontend Developer Guide][1].
```
Comment (hickskenh) [inline src/design/themes.md:11]: the link opens the guide at the top level.   the paragraph seems to be theme oriented.  consider using a link to the Theme topics.
https://devdocs.magento.com/guides/v2.3/frontend-dev-guide/themes/theme-overview.html

**Review by hickskenh — APPROVED** [review]
approved after your review of comments.

File: src/design/merge-css.md (hunk 1 above)
Comment (leslietilling) [reply src/design/merge-css.md:14]: this is the panel and doesn't use the same icon as the page.

File: src/design/merge-javascript.md (hunk 2 above)
Comment (leslietilling) [reply src/design/merge-javascript.md:14]: same

File: src/design/page-layout.md (hunk 3 above)
Comment (leslietilling) [reply src/design/page-layout.md:27]: I took it out there, I'll do that here too.

File: src/design/theme-assets.md (hunk 4 above)
Comment (leslietilling) [reply src/design/theme-assets.md:18]: same... no icon in panel

File: src/design/theme-install.md (hunk 5 above)
Comment (leslietilling) [reply src/design/theme-install.md:24]: will do

File: src/design/themes.md (hunk 6 above)
Comment (leslietilling) [reply src/design/themes.md:11]: will do

**Comment by leslietilling** [comment]
running tests

**Pull request merged**